frontend/src/components/modalContent/ViewIncidentModal.js

```diff
@@ -0,0 +1,288 @@
+          </p>
+        </div>
+      </div>
+      <br />
+    </div>
+  );
+
+  renderInitialDocuments = () => {
+    const initialDocuments = this.props.incident.documents.filter(
+      ({ mine_incident_document_type_code }) => mine_incident_document_type_code === "INI"
+    );
+    return (
+      <div>
+        <h5>Initial Report Documents</h5>
+        <Table
+          align="left"
+          pagination={false}
+          columns={this.columns()}
+          locale={{ emptyText: "This incident does not contain any initial documents" }}
+          dataSource={this.transformRowData(initialDocuments)}
+        />
+        <br />
+      </div>
+    );
+  };
+
+  renderFollowUpInformation = () => (
+    <div>
+      <h5>Follow-up information</h5>
+      <div className="content--light-grey padding-small">
+        <div className="inline-flex padding-small">
+          <p className="field-title">Was there a follow-up inspection?</p>
+          <p> {this.props.incident.followup_inspection ? "Yes" : "No"}</p>
+        </div>
+        <div className="inline-flex padding-small">
+          <p className="field-title">Follow-up inspection date</p>
+          <p>{formatDate(this.props.incident.followup_inspection_date) || Strings.EMPTY_FIELD}</p>
+        </div>
+        <div className="inline-flex padding-small">
+          <p className="field-title">Was it escilated to EMPR Investigation?</p>
+          <p>
+            {this.props.incidentFollowupActionHash[
+              this.props.incident.followup_investigation_type_code
+            ] || Strings.EMPTY_FIELD}
+          </p>
+        </div>
+        <div className="padding-small">
+          <p className="field-title">Mine managers recommendations</p>
+          {this.props.incident.recommendations.length >= 1 ? (
+            <div className="inline-flex">
+              <div className={this.state.recommendationsExpanded ? "block" : "collapsed-container"}>
+                {this.props.incident.recommendations.map(({ recommendation }) => (
+                  <p>- {recommendation}</p>
+                ))}
+              </div>
+              <Button className="btn--expand" onClick={() => this.toggleRecommendations()}>
+                {this.state.recommendationsExpanded ? "see less" : "... see more"}
+              </Button>
+            </div>
+          ) : (
+            <p>{Strings.EMPTY_FIELD}</p>
+          )}
+        </div>
+        <div>
+          <div className="inline-flex padding-small">
+            <p className="field-title">Incident Status</p>
+            <p>
+              {this.props.incidentStatusCodeHash[this.props.incident.status_code] ||
+                Strings.EMPTY_FIELD}
+            </p>
+          </div>
+        </div>
+      </div>
+      <br />
+    </div>
+  );
+
+  renderFinalDocuments = () => {
+    const finalDocuments = this.props.incident.documents.filter(
+      ({ mine_incident_document_type_code }) => mine_incident_document_type_code === "FIN"
```

Comment:
Same thoughts here.

I wonder if we may want to make an object of potential document types?
```javascript
const DOCUMENT_TYPES = {
  initial: "INI",
  final: "FIN",
};

const finalDocuments = this.props.incident.documents.filter(
  ({ mine_incident_document_type_code }) => mine_incident_document_type_code === DOCUMENT_TYPES.final
);
```

Compared to string constants, this might be the more reasonable way of maintaining our FE collection of valid codes from BE code tables 🤔 

Thoughts? Feelings?

Reply:
Like them being in constants. Meant to update that. 

We have similar constants for Variances that live in `strings.js` since they are used in multiple places. I'll initiate the constants in this file for now, as they aren't being used elsewhere.